relay-filter/src/transaction_name.rs

```diff
@@ -2,18 +2,24 @@
 //!
 //! If this filter is enabled transactions from healthcheck endpoints will be filtered out.
 
+use relay_common::EventType;
 use relay_general::protocol::Event;
 
 use crate::{FilterStatKey, GlobPatterns, IgnoreTransactionsFilterConfig};
 
 fn matches(event: &Event, patterns: &GlobPatterns) -> bool {
+    if event.ty.value() != Some(&EventType::Transaction) {
+        return false;
+    }
+
```

Comment:
Could we add the unit test at least to make sure this won't happen again?

Reply:
Will add test, will also fix the currently failing tests (that do not have the event type set)